streams/src/test/java/org/apache/kafka/streams/processor/internals/RecordContextStub.java

```diff
@@ -19,14 +19,18 @@
 public class RecordContextStub implements RecordContext {
 
     private final long offset;
-    private final long timestamp;
+    private long timestamp;
     private final int partition;
     private final String topic;
 
     public RecordContextStub() {
         this(-1, -1, -1, "");
     }
-    public RecordContextStub(final long offset, final long timestamp, final int partition, final String topic) {
+
+    public RecordContextStub(final long offset,
```

Comment:
In a different PR: could we replace this class with `MockProcessorContext` or `NoOpProcessorContext` below?

Reply:
I think so. \cc @vvcephei Can you include this in you MockProcessorContext PR ?